public_html/wp-content/mu-plugins/blocks/package.json

```diff
@@ -30,7 +30,8 @@
 		"css-loader": "1.0.1",
 		"lodash": "4.17.11",
 		"mini-css-extract-plugin": "0.4.4",
-		"node-sass": "4.9.4",
+		"node-sass": "4.11.0",
+		"rememo": "3.0.0",
```

Comment:
Is `rememo` still needed? Since switching to the Core selectors, it looks like maybe we're not using it anymore?

Reply:
Not needed anymore. will remove it.

Reply:
removed now. resolving conversation